packages/components/src/styles/Tooltip.module.css

```diff
@@ -17,10 +17,6 @@
 	isolation: isolate;
 	font: var(--lp-text-body-2-regular);
 	padding: var(--lp-spacing-200) var(--lp-spacing-300);
-	display: flex;
```

Comment:
Would you mind also adding `overflow: auto`? Generally, I think tooltips should be allowed to scroll?